src/sources/tripadvisor/reviews/models.rs

```diff
@@ -23,6 +23,10 @@ pub struct Review {
     pub date: Option<String>,
     #[serde(rename = "ReviewURL")]
     pub url: Option<String>,
+    #[serde(rename = "MoreReviewsURL")]
+    pub more_reviews_url: Option<String>,
```

Comment:
This review link feels weird as it is tied to a specific review where we'd want something that works for the whole POI. On the other hand I don't find anything similar in the property feed so I suppose Idunn would select one of those links for the "read more reviews on tripadvisor" button.